docs/contributing/style_guide.md

```diff
@@ -174,6 +174,25 @@ export interface PWrite {
 export function pwrite(options: PWrite) {}
 ```
 
+### Export all interfaces that are used as parameters to an exported member
+
+Whenever you are using interfaces that are included in the arguments of an exported member, you must export the interface that is used.  Here is an example:
+
+```ts
+// my_file.ts
+export interface Person {
+  name: string;
+  age: number;
+}
+
+export function createPerson(name: string, age: number): Person {
+  return { name, age };
+}
+
+// mod.ts
+export { Person, createPerson } from "./my_file.ts";
```

Comment:
In order to support `--no-check` this needs to be:

```ts
export { createPerson } from "./my_file.ts";
export type { Person } from "./my_file.ts";
```

Reply:
nit, also should be alphabetical

Reply:
> nit, also should be alphabetical

Sorry if I am just being slow, but what is nit?  Do you mean the interface labels?

Reply:
Nit as in "nitpick".

Reply:
> nit, also should be alphabetical

Thanks @caspervonb for clearing what "nit" means.  I usually order my imports in the order that I use them, but I will start using your convention👍

Reply:
[Yes, sorry](https://www.urbandictionary.com/define.php?term=nit)